### PR TITLE
Fix freedraw

### PR DESCRIPTION
**Pull request by Eirmas**
Fixed the freedraw so that it actually draws a line instead of small dots. Still need some work

### CODE REVIEW

**Comment by NikoGrano** [comment]
@Eirmas you fucked up. I'm not able to rebase. Please create new branch with only cherry picked commits and then create new pr for it.

File: src/mixins/CanvasTools.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class CanvasTools extends Vue {
         const line = new Konva.Line({
           stroke: tool.colour,
           strokeWidth: Number(tool.size),
-          lineJoin: 'round',
+          lineJoin: 'butt',
```
[the remark this answers is not in the document]
Comment (Eirmas) [reply src/mixins/CanvasTools.ts]: Dont know if there is a difference. Was just experimenting with the options and forgot to change back.

File: package.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
     "vue-router": "^3.0.6",
     "vue-socket.io-extended": "^4.0.1",
     "vuex": "^3.1.2",
-    "vuex-class": "^0.3.2"
+    "vuex-class": "^0.3.2",
+    "yarn": "^1.22.0"
```
[the remark this answers is not in the document]
Comment (Eirmas) [reply package.json]: Did not notice this. I didnt change that manually

File: src/mixins/CanvasTools.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (NikoGrano) [reply src/mixins/CanvasTools.ts]: 😂🤦‍♀️

File: src/components/canvas-tools/templates/Freedraw.vue [hunk 3]
```diff
@@ -68,10 +68,8 @@ export default class PopoutButton extends Vue {
 }
 
 </script>
-<style scoped lang="scss">
 
-</style>
-<style lang="scss">
+<style scoped lang="scss">
```
[the remark this answers is not in the document]
Comment (Eirmas) [reply src/components/canvas-tools/templates/Freedraw.vue]: It was scoped from the beginning. I just removed one styletag cause there were duplicates

**Pull request closed without merging**